docs/source/diagnosis.md

```diff
@@ -0,0 +1,64 @@
+# Quantization diagnosis
+There are 2 ways to diagnose quantized model with Intel® Neural Compressor. First is non-GUI mode that is described below and second is GUI mode with [Neural Insights](https://github.com/intel/neural-compressor/tree/master/neural_insights) component.
```

Comment:
I agree that in this docs we only need to introduce the details of non-GUI diagnosis, but let's put the workflow graph, so we can give an overall picture to the users.